### PR TITLE
Test: add test to IOServer

### PR DESCRIPTION
**Pull request by MrRuanCoder**
Just add new test to IOServer class and some maven dependencies（mockito and junit） in this pull request.
- Linked issue： #1

### CODE REVIEW

File: JMobileSuitLite.iml
Comment (FerdinandSukhoi) [inline JMobileSuitLite.iml:1]: As a maven project, naturally **JMobileSuitLite** should not include any IDE-related files (`.classpath, .project,.iml, .settings`, etc.).
I forgot to remove `.iml` files in previous versions, could you please help me remove all of them?
1. Add all the files to `.gitignore` file
2. Use `git rm --cached <file-pattern>` to remove tracked files

File: pom.xml
Comment (FerdinandSukhoi) [inline pom.xml:1]: I'm confused as to why there're two `junit` dependencies?

**Review by FerdinandSukhoi — CHANGES_REQUESTED** [review]
The tests look good,  but some changes are required. What's more, could you please provide the test results with coverage?

File: pom.xml [hunk 1]
```diff
@@ -10,6 +10,34 @@
             <artifactId>Jarvis</artifactId>
             <version>LATEST</version>
         </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
```
Comment (FerdinandSukhoi) [inline pom.xml:15]: I'm confused as to why there're two junit dependencies?

File: src/main/java/PlasticMetal/JMobileSuitLite/IO/IOServer.java [hunk 2]
```diff
@@ -355,6 +355,14 @@ public void SetPrefix(String value)
 
     private final StringBuilder PrefixBuilder = new StringBuilder();
 
+    public StringBuilder getPrefixBuilder() {
+        return PrefixBuilder;
+    }
+
```
Comment (FerdinandSukhoi) [inline src/main/java/PlasticMetal/JMobileSuitLite/IO/IOServer.java:361]: These members are not expected to be exposed to the callers, so they should not be Public members. Please consider using the default access level.

Comment (MrRuanCoder) [reply src/main/java/PlasticMetal/JMobileSuitLite/IO/IOServer.java:361]: OK, I will review it and  resubmit the pull request.

**Comment by MrRuanCoder** [comment]
coverage
Here are test coverage report

**Comment by MrRuanCoder** [comment]
[htmlReport.zip](https://github.com/Plastic-Metal/JMobileSuitLite/files/13346073/htmlReport.zip)

**Pull request merged**